python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1019,7 +1019,8 @@ async def move_camera_hexapod(
     ) -> None:
         """Move camera hexapod.
 
-        When camera hexapod compensation mode is on move will act as offset.
+        When camera hexapod compensation mode is on, move is with respect to
+        the LUT value.
```

Comment:
Could we add some extra documentation here to clarify what happens if the compensation mode is not enabled? I guess that it moves to an absolute value. If this is the case, maybe something like this?
```
When the camera hexapod compensation mode is on, 
move it to a new position relative to the LUT. 
When the camera hexapod compensation mode is off, 
move it to a new absolute position. 
```